### PR TITLE
Add JPY to list of supported currencies 

### PR DESCRIPTION
**Pull request by mrbkiter**
Now Recurly supports JPY (Japanese currency).

### CODE REVIEW

**Comment by bhelx** [comment]
@mrbkiter I've been a bit busy. I'll be checking this in the next couple days and hopefully we can get it in this week. Thanks a bunch for contributing JPY!

**Comment by mrbkiter** [comment]
ok thanks @bhelx Let me know if you need anything

File: src/main/java/com/ning/billing/recurly/model/RecurlyUnitCurrency.java [hunk 1]
```diff
@@ -320,27 +330,31 @@ public boolean equals(final Object o) {
...
-        result = 31 * result + (unitAmountHUF != null ? unitAmountHUF.hashCode() : 0);
-        result = 31 * result + (unitAmountNOK != null ? unitAmountNOK.hashCode() : 0);
-        result = 31 * result + (unitAmountNZD != null ? unitAmountNZD.hashCode() : 0);
-        result = 31 * result + (unitAmountPLN != null ? unitAmountPLN.hashCode() : 0);
-        result = 31 * result + (unitAmountSGD != null ? unitAmountSGD.hashCode() : 0);
-        result = 31 * result + (unitAmountSEK != null ? unitAmountSEK.hashCode() : 0);
-        result = 31 * result + (unitAmountCHF != null ? unitAmountCHF.hashCode() : 0);
-        result = 31 * result + (unitAmountZAR != null ? unitAmountZAR.hashCode() : 0);
-        return result;
+        return Objects.hashCode(
```
Comment (bhelx) [inline src/main/java/com/ning/billing/recurly/model/RecurlyUnitCurrency.java]: :+1: Thanks for this refactor

**Comment by bhelx** [comment]
@mrbkiter The code looks good to me. It should be fine to merge. Do you mind rebasing with master and resolving the conflicts?

File: src/main/java/com/ning/billing/recurly/model/RecurlyUnitCurrency.java (hunk 1 above)
Comment (mrbkiter) [inline src/main/java/com/ning/billing/recurly/model/RecurlyUnitCurrency.java]: It should be from current master :-)

**Comment by bhelx** [comment]
@mrbkiter I can't merge it until you rebase with killbill:master. Make sure your upstream repo is up to date.

**Comment by bhelx** [comment]
@mrbkiter any updates?

**Comment by bhelx** [comment]
@mrbkiter let me know if you are having trouble, i'd be happy to help

**Comment by mrbkiter** [comment]
Hi @bhelx 

Sorry for my late reply. I was super busy recent weeks. 

Let me do the merge today. Thanks!

**Comment by mrbkiter** [comment]
I resolved conflict. Is it ok for rebasing now, @bhelx ?

**Comment by bhelx** [comment]
@mrbkiter It looks good now. Going to run the tests real quick.

**Comment by bhelx** [comment]
@mrbkiter Okay looks good :+1:

**Comment by bhelx** [comment]
@mrbkiter Thanks a lot for contributing this change!

**Comment by mrbkiter** [comment]
You're welcome, @bhelx

**Pull request merged**